Functions.Templates/Templates/HttpTrigger-PowerShell/run.ps1

```diff
@@ -1,24 +1,28 @@
+using namespace System.Net
+
 # Input bindings are passed in via param block.
-param($req, $TriggerMetadata)
+param($Request, $TriggerMetadata)
 
 # Write to the Azure Functions Trace-level log stream.
 Write-Verbose "PowerShell HTTP trigger function processed a request." -Verbose
```

Comment:
I just realized that we're using Verbose here but the default `host.json` that gets created with the command `func init` does not turn on Verbose logging.

I think we should either:

* change this log call to Information or Warning
* _add_ a call to Information or Warning and include a comment on how to turn on Verbose logging.

Tangent:
There are some [docs here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json#logging) on how to configure logging... But they're kind limited. Also there's a disconnect between Functions and PowerShell...

PowerShell uses "Verbose"
Functions uses "Trace"

I've [opened an issue](https://github.com/Azure/azure-functions-host/issues/3701) about that disconnect.

Reply:
Information makes more sense than Warning.  We should agree on the best practice and apply to all our templates.

Reply:
I'm fine with Information. Host also goes through as Information but I know we say Host is not a good choice so Information is what we should go with IMO.

So long as the comment above it says that you can use the same write streams that you expect in PowerShell.

Reply:
Based on team discussion, we decided to use `Write-Host` in our templates